src/config/configuration.ts

```diff
@@ -58,6 +58,10 @@ const configuration = () => {
   });
 
   const config = {
+    VERSIONS: {
+      API: "v3",
+    },
```

Comment:
Do you mind using camel case, so we are consistent with all the other entries?

Reply:
fixed